client/src/components/Home.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import { useEffect } from "react";
+import css from "../App.css";
+import HomeCss from "./HomeCss.css"
+
+const Home = () => {
+    const [joins, setJoins] = useState([]);
+
+    //get joined data table
+    const getJoins = async () => {
+        const response = await fetch(
+            `http://localhost:4040/joined`
+        );
+        const join = await response.json();
+        console.log(join);
+        setJoins(join);
+    };
+
+    useEffect(() => {
+        getJoins();
+    }, []);
+
+    const [searchTerm, setSearchTerm]=useState('');
+
+    return (
+        <>
+            <header> List all Sightings of Individuals Data Table </header>
+            <br></br>
+            <input type="text"
+                placeholder="Search..."
+                className="search"
+                onChange={(e) => setSearchTerm(e.target.value)} />
+            <br></br>
+            <table>
+                <thead>
+                    <td>ID: </td>
+                    <td>Date Time: </td>
+                    <td> Location:</td>
+                    <td> Healthy:</td>
+                    <td>Individual Id:</td >
+                    <td> Created On:</td >
+                    <td> Email:</td >
+                    <td> Nickname:</td >
+                    <td> Seen On:</td >
+                    <td> Species Id:</td >
+            </thead >
+    <tbody>
+        {joins.filter((val) => {
+            if (searchTerm === '') {
+                return val;
+            } else if (val.location.toString().toLowerCase().includes(searchTerm.toLowerCase())) {
```

Comment:
This looks pretty good. You could also do this in the db either with [postgres text search](https://www.postgresql.org/docs/current/textsearch-intro.html#TEXTSEARCH-MATCHING) or the more basic operators like ~, ~*, LIKE, and ILIKE.

Reply:
The advantage would be the fe would be much simpler here and you can search over the whole set of species without loading them all in the fe.